src/cli/plugin/plugin_lister.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+module.exports = {
```

Comment:
Okay, will make this change in this file. Should I also make the same change in the other `plugin_*.js` files in this directory (that aren't in the scope of this PR)?